automobile/scenario-creation-tutorial.md

```diff
@@ -96,6 +44,34 @@ You should be able to open the generated world file directly in Webots:
 ![osm_tutorial_import2.png](images/osm_tutorial_import2.png)
 %end
 
+It is recommended at this stage to perform a manual check of the imported roads and crossroads,
+to erase possible issues done by the importer.
+A good strategy could be fix the OSM data typically to add/remove wayPoints and import again the simulation.
+
+
+### Generate the SUMO network files
+
+We can also use the previously downloaded map to generate the SUMO network file. You need to use the [netconvert](http://sumo.dlr.de/wiki/NETCONVERT) utility for this:
```

Comment:
This sentence seems wrong, you don't really use the downloaded map, but the generated world.